src/library_browser.js

```diff
@@ -264,6 +264,48 @@ var LibraryBrowser = {
           }, false);
         }
       }
+
+      function Fullscreen (testElement) {
```

Comment:
I'd prefer we don't add new kind of stateful abstractions like this. For avoiding code duplication e.g. on obtaining the current fullscreen element, prefer to have a new function that would read something like

    function fullscreenElement() {
        return document['fullScreenElement'] || document['fullscreenElement'] ||
            document['mozFullScreenElement'] || document['mozFullscreenElement'] ||
            document['msFullScreenElement'] || document['msFullscreenElement'] ||
            document['webkitFullScreenElement'] || document['webkitFullscreenElement'] ||
            document['webkitCurrentFullScreenElement'];
    }

and then call that everywhere we need to get the fullscreen element. That is easier to follow than relate to, than adding wrapper over the fullscreen API.